contrib/linux/install-icons.sh

```diff
@@ -25,14 +25,16 @@ usage() {
 
 install_icons() {
     cp -r icons $DEST_PATH
-    cp desktop/dosbox-staging.desktop $DEST_PATH/applications
+    cp desktop/org.dosbox-staging.dosbox-staging.desktop $DEST_PATH/applications/org.dosbox-staging.dosbox-staging.desktop
 
     ## Replace executable location in the .desktop file
-    sed -i 's\Exec=dosbox\Exec='$(realpath dosbox)'\g' $DEST_PATH/applications/dosbox-staging.desktop
+    sed -i 's\Exec=dosbox\Exec='$(realpath dosbox)'\g' $DEST_PATH/applications/org.dosbox-staging.dosbox-staging.desktop
 }
 
 remove_icons() {
     find $DEST_PATH/icons -name dosbox-staging* -delete
```

Comment:
Same comment I made on the original PR.  This no longer deletes icons since the names have changed.

Reply:
Fair point.

Reply:
I think I misunderstood the comment - to fix the problem I have added lines:

```bash
    ## Also remove legacy files, if present
    rm $DEST_PATH/applications/dosbox-staging.desktop
```

to the `remove_icons()` function. AFAIK this `sed` command does not remove eny files, it just updates the content of the installed `.desktop` file. Shall I remove legacy files here, too? I don't really know what are the exactl use cases of this script, unfortunately.

Reply:
The script is meant to be run by end users that download the tarball from our website.  It copies the icon files and .desktop file to either `~/.local/share/` or, with the system flag `/usr/share/applications`.  This installs a shortcut and is also needed by Wayland to show the Staging icon in the task bar (Wayland doesn't have a way to programmatically set this unlike X11 so it looks for a .desktop file that matches the executable).

These files are re-named in this PR so the remove function will not work.  You've got the .desktop part handled correctly but not the .png files.

Reply:
OK, now I understand. Fixed.